tests/test_samples.py

```diff
@@ -395,6 +395,46 @@ def test_hist_levels():
                levels=[0.95, 0.68], bins=20)
 
 
+def test_mcmc_stats():
+    mcmc = read_chains('./tests/example_data/cb')
+    chains = mcmc.groupby(('chain', '$n_\\mathrm{chain}$'), group_keys=False)
+    n0 = chains.count().iloc[0, 0]  # number samples in first chain
+    mcmc_head = chains.head(200).copy()
+    mcmc_asym = mcmc.remove_burn_in(burn_in=[2, 10])
```

Comment:
Should we test asymmetric floats as well as integers?

Reply:
I've heavily expanded on the tests to cover the various possible inputs (see https://github.com/williamjameshandley/anesthetic/pull/258/commits/95b90263f39ab038206cedf6bf93a838de580ea1). 

I've even included negative inputs (which effectively function as a "keep" rather than a "remove" --> also added to the docstring in https://github.com/williamjameshandley/anesthetic/pull/258/commits/a25117f66dc733f00ce62beca31d135379b2a3dc).